.github/workflows/report-maker.yml

```diff
@@ -196,3 +199,52 @@ jobs:
           No ${{ steps.setup2.outputs.error_name }}! :tada:
           _Comment updated at ${{ steps.build-components2.outputs.time }} with changes from ${{ steps.build-components2.outputs.commit_id }}_
         edit-mode: replace
+
+  sort-dictionary:
+    runs-on: ubuntu-latest
+    if:  inputs.sort_dictionary && inputs.check_type == 'spelling'
+    steps:
+    - name: "Check out PR branch"
+      id: checkout-pr-branch
+      uses: actions/checkout@v4
+
+    - name: "Check write permissions"
+      id: check-write-permissions
+      run: |
+         sudo apt-get install -y jq
```

Comment:
Ah this is good. We should make this a function and use this elsewhere because permissions (or lack thereof) is a frequent issue for ottr users.